src/modules/navigator/navigator.h

```diff
@@ -157,6 +159,8 @@ class Navigator : public ModuleBase<Navigator>, public ModuleParams
 	struct vehicle_status_s *get_vstatus() { return &_vstatus; }
 	PrecLand *get_precland() { return &_precland; } /**< allow others, e.g. Mission, to use the precision land block */
 
+	struct vehicle_trajectory_waypoint_s *get_trajectory_waypoint() { return &_traj_wp_avoidance;}
```

Comment:
is it a good pattern to return pointers to private members? I see the same on lines below and above so it might not be something that should be fixed in this PR

Reply:
I believe almost all of these can be changed to return const references fairly easily.